### PR TITLE
Styling fixes and improvements

### PR DESCRIPTION
**Pull request by snurrfint**

### CODE REVIEW

File: src/components/table-field.pcss [hunk 1]
```diff
@@ -22,15 +22,23 @@
...
+      font-weight: 600;
+      font-size: 1.5em;
+    }
   }
 
   &.filtered {
-    box-shadow: inset 0 0 2px #0f0;
+    .name {
+      background-color: #019b39;
+      padding: 0 .25em;
```
Comment (axelssonHakan) [inline src/components/table-field.pcss]: I would like a bit more left/right padding

**Comment by hrigner** [comment]
The first two filterboxes in the selection toolbar wont open..

File: src/components/table-field.pcss (hunk 1 above)
Comment (snurrfint) [reply src/components/table-field.pcss]: .5em, thats all I can give you ;)

Comment (axelssonHakan) [reply src/components/table-field.pcss]: I would have settled on .4

**Pull request merged**